recipes/nnetsauce/meta.yml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nnetsauce" %}
```

Comment:
change file name to `meta.yaml` instead of `meta.yml`

Reply:
meta.yaml instead of meta.yml ??

Reply:
yes, please rename this file `meta.yaml`, but keep it in the directory it is currently in.

Reply:
done